utils/s3/client.hh

```diff
@@ -36,6 +38,7 @@ class client : public enable_shared_from_this<client> {
     struct private_tag {};
 
     void authorize(http::request&);
+    future<> make_request(http::request req, http::experimental::client::reply_handler handle = ignore_reply, http::reply::status_type expected = http::reply::status_type::ok);
 
```

Comment:
I asked myself if ignore_reply should be the default handler. ignore_reply sounds something that the client user will explictly tell the request marker. I understand the goal is to remove verbosity, but maybe it's not a good idea?

Reply:
maybe add make_request_with_reply_ignored?

Reply:
The `ignore_reply` just means that the caller doesn't want to read reply body back.  Status code matters in any case

Reply:
So maybe should be called ignore_reply_body, or something?

Reply:
+1 for a more specific name, as my first reaction was that ignore_reply could be about ignoring the whole reply, which doesn't make sense given the context, but I think we can benefit from having a more specific name here, for improved readability.